libs/sr/page-team/src/Optimize/GeneratedBuildsDisplay.tsx

```
@@ -0,0 +1,162 @@
+                }}
+              >
+                <Typography>
+                  Build {i + 1}: {valueString(build.value)}
+                </Typography>
+                <NewBuildButton build={build} />
+              </Box>
+
+              <GeneratedBuildDisplay build={build.relicIds} />
+            </CardContent>
+          </CardThemed>
+        ))}
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)